src/ovro_lwa_portal/viz/explorers.py

```
@@ -0,0 +1,452 @@
+    """
+
+    l_val = param.Number(default=0.0, bounds=(-1.2, 1.2), step=0.01, doc="l direction cosine")
+    m_val = param.Number(default=0.0, bounds=(-1.2, 1.2), step=0.01, doc="m direction cosine")
+    cmap = param.Selector(default="inferno", objects=COLORMAPS, doc="Colormap")
+    robust = param.Boolean(default=True, doc="Use robust color scaling")
+
+    def __init__(
+        self,
+        ds: xr.Dataset,
+        *,
+        l: float | None = None,
```

Ruff (0.15.10)

[error] Ambiguous variable name: `l`

(E741)